README.md

```diff
@@ -1,5 +1,6 @@
 # Azimuth
 
+[![Build Status](https://secure.travis-ci.org/urbit/azimuth.png)](http://travis-ci.org/urbit/azimuth)
```

Comment:
I never understood the point of these. Github already shows status, and we don't want to merge non-passing versions anyway.

Reply:
> Uh... probably don't do this? Nothing but the Ecliptic can be upgraded, and even there you want to keep a copy of the old version at Ecliptic1.sol or w/e. The contracts here should match their on-chain versions, and those just can't change.

Yeah, I figured we would only propose to merge that work conjointly with a proposal to the senate to upgrade the ecliptic (the old version would remain in the git history, of course).

For the others -- I just sort of lazily intuited that new versions of at least most could be deployed and ownership/permissions/etc. reassigned as necessary, appealing to the senate as needed to approve the required changes.  Is that not true?

It seems like it would be nice to be able to use more recent solc, openzeppelin-contracts, etc. versions for potential security/gas cost improvements, but perhaps muh endeavour is misguided..